about/changelog.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Changelog
+excerpt: Changelog
+keywords: [changelog, upgrades, updates, releases]
+---
+
+# Changelog
+
+## TimescaleDB 2.15.2
+
+June 7, 2024
+
+🐛 Bugfixes
+
+- [#6975](https://github.com/timescale/timescaledb/issues/6975): Fix sort pushdown for partially compressed chunks.
+- [#6976](https://github.com/timescale/timescaledb/issues/6976): Fix removal of metadata function and the update script.
+- [#6978](https://github.com/timescale/timescaledb/issues/6978): Fix segfault in compress_chunk with primary space partition.
+- [#6993](https://github.com/timescale/timescaledb/issues/6993): Disallow hash partitioning on the primary column.
+
+## May 2024 Update
+
+May 31, 2024
+
+### 🔍 Database audit logging via the pg_audit extension
+
+The [PostgreSQL Audit Extension (pgaudit)](https://github.com/pgaudit/pgaudit/) is now available on Timescale. It provides detailed database session and/or object audit logging in the Timescale logs.
+
+If you have strict security and compliance requirements and you need to log all operations on the database level, pgaudit can help. You can also export audit logs to CloudWatch if you like.
+
+To install the extension, simply go to `Service > Operations > Extensions` and search for `pgaudit`. To change the configuration parameters, go to `Database parameters > Advanced parameters` and search for `pgaudit`.
+
+For more details on how to use the extension, please refer to the [pgaudit documentation](https://github.com/pgaudit/pgaudit/).
+
+### 🌡 Timescale now supports the unit extension :thermometer:
+
+Timescale now also supports the [unit](https://github.com/df7cb/postgresql-unit) extension.
+
+This extension provides SI Unit support in Postgres. This means that you can use Postgres to solve some of the more annoying day-to-day questions in your life.
```

Comment:
At a glance this confused me, I read these examples as returning weather and fish data rather than comparing units. 

[postgresql-unit](https://github.com/df7cb/postgresql-unit) provides support for the [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units) inside your Timescale Cloud service. You can use Timescale Cloud to solve day-to-day questions. For example, to see what 50°C is in °F, run the following query in your Timescale Cloud service: 

```
SELECT '50°C'::unit @ '°F' as temp;
  temp
--------
 122 °F
(1 row)
```

Reply:
f98017435ba80df07a61284029faef7a7eec296a